assets/css/homepage.css

```diff
@@ -0,0 +1,89 @@
+html, body {
+     height: 100%;
+     width: 100%;
+     margin: 0;
+     padding: 0;
+     font-family: 'Inter', sans-serif;
+ }
+ 
+ body {
+     background-image: url('../img/background.jpg');
+     background-size: cover;
+     background-position: center;
+     background-repeat: no-repeat;
+     background-attachment: fixed;
+ }
+ 
+ main {
+     padding: 20px;
+     display: flex;
+     justify-content: center;
+ }
+ 
+ .product-list {
+     background-color: rgba(255, 255, 255, 0.9);
+     padding: 20px;
+     border-radius: 10px;
+     box-shadow: 0px 4px 50px 0px rgba(0, 0, 0, 0.25);
+     width: 100%;
+     max-width: 1804px;
+ }
+ 
+ .product-list h2 {
+     text-align: left;
+     font-size: 2rem;
+     font-weight: 700;
+     color: #222222;
+     margin: 15px 5px;
+ }
+ 
+ .product-grid {
+     display: grid;
```

Comment:
> [!TIP]
> For this project it's Okay to use `grid` as your display and there's no need to change it. but in more popular apps, they usually try to avoid this kind of property, because some older browsers do not support them. you can use [caniuse](https://caniuse.com) to find out which browsers can support what you are including in your code